src/argilla/client/feedback/training/frameworks/transformers.py

```diff
@@ -57,9 +57,9 @@ def __init__(self, *args, **kwargs):
         else:
             raise NotImplementedError(f"We do not support {type(self._dataset)} yet.")
 
-        if isinstance(self._task_mapping, TrainingTaskMappingForTextClassification):
+        if isinstance(self._task, TrainingTaskForTextClassification):
             self._model_class = AutoModelForSequenceClassification
         else:
-            raise NotImplementedError(f"ArgillaTransformersTrainer does not support {type(self._task_mapping)} yet.")
+            raise NotImplementedError(f"ArgillaTransformersTrainer does not support {type(self._task)} yet.")
```

Comment:
Maybe?

```suggestion
            raise NotImplementedError(f"ArgillaTransformersTrainer does not support {self._task.__name__} yet.")
```

Reply:
I think there's a chance that this bit is dead code, but I'll use `self._task.__class__.__name__`